index.html

```diff
@@ -376,113 +396,203 @@ <h4>Characters</h4>
+  &lt;/ttm:agent&gt;
+  &lt;ttm:agent type=&quot;person&quot; xml:id=&quot;actor_b89ea6a2-bcfa-46b4-a835-fc13081672e7&quot;&gt;
+    &lt;ttm:name type=&quot;full&quot;&gt;Matthias Schoenaerts&lt;/ttm:name&gt;
+  &lt;/ttm:agent&gt;
+&lt;/medadata&gt;
+...
+        </pre>
 
-      <section>
-        <h3>Event</h3>
-        <p>An <dfn>Event</dfn> has the following properties.</p>
+            <li>If the <a>Character</a> has a <a>Style</a>, the following TTML constraints apply:
+              <ul>
+                <li>There MAY be a <code>style</code> element in the <code>styling</code> element providing the styles applicable to <a>Events</a> from this <a>Character</a>. The <code>xml:id</code> MUST be present and represents the <a>Character Identifier</a>. Any style attribute from [[ttml-imsc1.1]] may be present. A <code>ttm:agent</code> attribute MAY be present to link the <code>ttm:agent</code> element representing the <a>Character</a>.</li>
+              </ul>
+            </li>
 
-        <section>
-          <h4>Event Identifier</h4>
-          <p>Each <a>Event</a> MUST have a unique <dfn>Event Identifier</dfn>.</p>
-        </section>
+        <pre class="example">
+...
+&lt;styling&gt;
+  &lt;style xml:id=&quot;style_0085df18-5020-4473-a312-fe4b233bad65&quot;
+         ttm:agent=&quot;character_0085df18-5020-4473-a312-fe4b233bad65&quot;
+         tts:color=&quot;#FFFFFF&quot; tts:backgroundColor=&quot;#8F42AD&quot;/&gt;         
+&lt;/styling&gt;     
+...
+        </pre>
 
-        <section>
-          <h4>Event Type</h4>
-          <p>Each <a>Event</a> in a Dubbing Script MAY have at most one <dfn>Event Type</dfn> used to identify if the <a>Event</a> represents spoken text or on screen text, and in the later case, the type of on-screen text (title, credit, location, ...).</p>
-        </section>
+          </ul>
+        </p>
 
-        <section>
-          <h4>Character Identifier</h4>
-          <p>Each <a>Event</a> in a Dubbing Script MUST have one or more identifiers of <a>Character</a> involved in this <a>Event</a>.</p>
-          <p class="note">While typically, an <a>Event</a> corresponds to one single character, there are cases where multiple characters can speak the same text at the same time.</p>
-        </section>
+        <p class="issue" data-number="29"></p>
+        <p class="issue" data-number="15"></p>
+      </section>
 
-        <section>
-          <h4>Start</h4>
-          <p>Each <a>Event</a> MUST have a <dfn>Start</dfn> property.</p>
-        </section>
+      <section>
+        <h3>Event</h3>
+        <p>An <dfn>Event</dfn> object represents dialogs, on screen text or audio descriptions to be spoken and has the following properties:
+          <ul>
+            <li>An <dfn>Event Identifier</dfn> which is unique in the script</li>
+            <li><dfn>Start</dfn> and <dfn>End</dfn> properties providing the <a>Event</a> time interval in the programme timeline
```

Comment:
I'm not sure what the confusion is.

Reply:
Since TTML uses `begin` and `end` and they are exactly the same as the meanings here, we should use `Begin` instead of `Start`. Otherwise it looks like we're defining something that's somehow different but the same, without any good reason, which is confusing.